packages/coingecko/recipes/e2e.json

```diff
@@ -4,6 +4,8 @@
   },
   {
     "query": "./get.graphql",
```

Comment:
This is just what I put together quickly while bootstrapping but we would like to have separate files for each queries.